client/modules/IDE/components/Console.jsx

```diff
@@ -39,16 +44,24 @@ class Console extends React.Component {
       LOG_DEBUG_ICON: `url(${debugDarkUrl})`,
       LOG_INFO_ICON: `url(${infoDarkUrl})`
     };
+    const CONSOLE_FEED_SIZES = {
+      BASE_FONT_SIZE: this.props.fontSize,
+      TREENODE_LINE_HEIGHT: 1.5,
+      ARROW_FONT_SIZE: this.props.fontSize,
+      LOG_ICON_WIDTH: `${this.props.fontSize}px`,
+      LOG_ICON_HEIGHT: `${this.props.fontSize}px`,
```

Comment:
lt's ok to use `this.props.fontSize` here, could you make these consistent with the above?

Reply:
I mean you don't need to use px here, right?

Reply:
oh, i putted px in above lines :sweat_smile:

Reply:
``` js
    const CONSOLE_FEED_SIZES = {
      TREENODE_LINE_HEIGHT: 1.5,
      BASE_FONT_SIZE: this.props.fontSize,
      ARROW_FONT_SIZE: this.props.fontSize,
      LOG_ICON_WIDTH: this.props.fontSize,
      LOG_ICON_HEIGHT: this.props.fontSize,
    };
```

is this ok or not ?